### DISCUSSION TITLE
basic-extra-meal/fanshe

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#209] [Announcements]
# basic-extra-meal/fanshe

Java 反射机制允许在运行时检查和操作类、对象、方法和字段。通过反射，我们可以动态创建对象实例、调用方法、访问字段和获取类的元数据。本文介绍了 Java 反射的基本概念、应用场景和示例。

https://tobebetterjavaer.com/basic-extra-meal/fanshe.html

**Comment by IDcardBull** [comment]
写的好

**Comment by szy0417-maker** [comment]
终于弄懂了反射，感谢
